src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
 public class SalaryInfo {
+    private static final String REPORT_HEADER = "Report for period ";
+    private static final String SEPARATOR = " - ";
+    private static final int DATE_POSITION = 0;
+    private static final int NAME_POSITION = 1;
+    private static final int HOUR_POSITION = 2;
+    private static final int MONEY_PER_HOUR_POSITION = 3;
+    private static final String SPACE = " ";
+    private static final DateTimeFormatter FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy", Locale.ENGLISH);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate parsedDateFrom = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate parsedDateTo = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder builder = new StringBuilder();
```

Comment:
Minor: I would suggest being more specific on this variable name and specifying the builder of what this variable actually stores